src/store/CrosshairStore.js

```diff
@@ -354,7 +355,7 @@ class CrosshairStore {
 
     // YES! we are manually patching DOM, Because we don't want to pay
     // for react reconciler & mox tracking observables.
-    updateTooltipPosition({ top, left, rows }) {
+    updateTooltipPosition({ top, left, rows, show = false }) {
```

Comment:
pass `top: -5000px, left: -5000px` instead, similar to how we do it here
https://github.com/binary-com/SmartCharts/pull/830/files#diff-2f67da90737005346bff0e111831096cR94